taskmanager: compare int with null

```
[WARNING] /Users/ark/Code/OpenMLDB/java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala:81: warning: comparing values of types Int and Null using `!=' will always yield true
[INFO]     if (TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS != null) {
```

--- a/java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala
+++ b/java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala
@@ -78,8 +78,7 @@ object SparkJobManager {
       launcher.setConf("spark.eventLog.dir", TaskManagerConfig.SPARK_EVENTLOG_DIR)
     }
 
-    if ((TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS) >= 1 ) {
-      setConf
+    if (TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS >= 1 ) {
       launcher.setConf("spark.yarn.maxAppAttempts", TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS.toString)
     }
 

--- a/java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala
+++ b/java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala
@@ -78,7 +78,7 @@ object SparkJobManager {
       launcher.setConf("spark.eventLog.dir", TaskManagerConfig.SPARK_EVENTLOG_DIR)
     }
 
-    if (TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS != null) {
+    if ((TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS : java.lang.Integer) != null) {
       launcher.setConf("spark.yarn.maxAppAttempts", TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS.toString)
     }
 

--- a/java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala
+++ b/java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala
@@ -78,7 +78,8 @@ object SparkJobManager {
       launcher.setConf("spark.eventLog.dir", TaskManagerConfig.SPARK_EVENTLOG_DIR)
     }
 
-    if ((TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS : java.lang.Integer) != null) {
+    if ((TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS) >= 1 ) {
+      setConf
       launcher.setConf("spark.yarn.maxAppAttempts", TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS.toString)
     }
 

